scrapers/SpankBang.yml

```diff
@@ -0,0 +1,31 @@
+name: "SpankBang"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - spankbang.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class='left']/h1/@title
+      Details: //meta[@name='description']/@content
+      Image: //div[@class='play_cover']/img/@src
+      Studio:
+        Name:
+          selector: //li[@class='us']/a[@class='ul']/@href[contains(.,'channel')]
```

Comment:
do we need the subscraper ?
it seems to me that 
```yaml
      Studio:
        Name: //li[@class='us']//a[@class='ul'][contains(@href,'channel')]
```
is enough

Reply:
You are correct! Fixing it